### PR TITLE
front: match the width and spacing of the results table to the mockup

### LINKED ISSUE #9886: Stdcm results table width no matching the mockup
### What happened?

App : 
Image

Mockup : 
Image

### What did you expect to happen?

- [ ] Both map should be aligned
- [ ] Table should be shrinked, @thibautsailly will give the column to resize and the values

### How can we reproduce it (as minimally and precisely as possible)?

Go to stdcm page and launch a simulation

### On which environments the bug occurs?

Local

### On which browser the bug occurs?

Firefox

### OSRD version (top right corner `Account` button > `Informations`)

.

### PR DESCRIPTION
**Pull request by theocrsb**
closes #9886

### CODE REVIEW

**Review by Akctarus — APPROVED** [review]
Tested, LGTM

**Review by clarani — COMMENTED** [review]
I still have 30px of padding-left on the map, I should have 32px according to the mockup. The rest of the code is fine for me :white_check_mark:

File: front/src/styles/scss/applications/stdcm/_results.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .stdcm-results {
   background-color: rgb(233, 239, 242);
-  padding: 29px 30px 32px 32px;
+  padding: 29px 32px 32px 32px;
```
Comment (clarani) [inline front/src/styles/scss/applications/stdcm/_results.scss]: I think you can do this :)
```suggestion
  padding: 29px 32px 32px;
```

**Review by clarani — APPROVED** [review]
LGTM :white_check_mark:

File: front/src/styles/scss/applications/stdcm/_results.scss (hunk 1 above)
Comment (theocrsb) [reply front/src/styles/scss/applications/stdcm/_results.scss]: Done

**Pull request merged**